documentation/writing_plans.md

```diff
@@ -1094,6 +1094,23 @@ MODULE:
 /Users/bolt/.puppetlabs/bolt
 ```
 
+## Making plans private
+
+You can hide plans from `bolt plan show` and `Get-BoltPlan` output by specifying the `# @private
+true` Puppet strings tag. Private plans are still viewable with `bolt plan show <plan_name>` and
+`Get-BoltPlan -Name <plan_name>`, and can still be run with Bolt.
```

Comment:
I think it would be good to add a "why" here. Why would I want to hide the plan from show output?